packages/dx-react-grid-material-ui/src/templates/table-edit-cell.jsx

```diff
@@ -13,9 +13,7 @@ const styles = theme => ({
     verticalAlign: 'top',
     paddingTop: theme.spacing.unit + 2,
```

Comment:
Why `+ 2`? Should not this value depend on the `theme.spacing.unit`? Or is this a borders related correction?

Reply:
> Or is this a borders related correction?

I think so.